src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/CreateActionPlanDTO.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import java.time.OffsetDateTime
+import java.util.UUID
+
+data class CreateActionPlanDTO(
+  val referralId: UUID,
+  val numberOfSessions: Int?,
```

Comment:
will these values (numberOfSessions and activities) be known at action plan creation time?

Reply:
Until the contract is defined I don't know either way.